knative-operator/deploy/resources/console_cli_download_kn.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: kn
+spec:
+  description: The OpenShift Serverless client `kn` is a CLI tool that allows you to fully manage OpenShift Serverless Serving and Eventing resources without writing a single line of YAML.
+  displayName: kn - OpenShift Serverless CLI
+  links:
+  - href: https://mirror.openshift.com/pub/openshift-v4/clients/serverless/0.11.0/kn-linux-amd64-0.11.0.tar.gz
```

Comment:
I wonder whether we should parameterize the version number at some point in time (and make it part of the build process to put that into the resource file). Maybe manifestival know how to process templates of that kind ? Then we could add a placeholder here and add the real version from within the operator itself.

Reply:
I'm open to env substitution in manifestival, but in this case it might be just as easy to munge the appropriate `manifest.Resources` to replace some token in the file with `instance.Status.Version` before invoking `manifest.ApplyAll`

Reply:
yeah, whatever :) Doing the replace ourselves is probably easier (only if manifestival would already have supported templating then we could have used that). But could do now that it's easy to get to the operator version (which actually also corresponds to the `kn` version).

@navidshaikh do you think this would be hard to implement ? we can do that post 1.4 though (as the current approach works)

Reply:
We can make it more parameterized, sure!
With air-gapped scenario, will have to re-work the ConsoleCLIDownload and associated artifacts management, will keep this in mind. Thanks for the review and suggestions!